app/models/tasks/copy_media_task.rb

```diff
@@ -48,18 +48,29 @@ def update_media_resource
         media_resource.duration = porter_callback_transcode[:Duration]&.to_f&./ 1000
       end
 
-      # change status, for invalid and bad-duration files
-      if media_resource.invalid?
-        media_resource.status = "invalid"
-      elsif info[:Audio].try(:[], :DurationDiscrepancy).to_i > 500
-        media_resource.status = "processing"
-        Tasks::FixMediaTask.create! { |t| t.owner = media_resource }.start!
-      end
+      # change status, if metadata doesn't pass validations
+      media_resource.status = "invalid" if media_resource.invalid?
```

Comment:
Reverted this to a couple PRs ago, and moved the "fix this file" logic down lower.  Functionally pretty similar, but reads a bit better.